libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationLifecycleMonitor.java

```diff
@@ -55,6 +55,7 @@ public void onActivityPaused(Activity activity) {
   @Override
   public void onActivityDestroyed(Activity activity) {
     if (activity.isFinishing()) {
+      NavigationTelemetry.getInstance().endSession(activity.isChangingConfigurations());
```

Comment:
@ericrwolfe Now we call `NavigationTelemetry#endSession` only if the activity is finishing and pass whether it is finishing from a changing config or not.